File: py/program/produk_elektronik.py
from produk import Produk

class ProdukElektronik(Produk):
    def __init__(self, id_produk, nama, deskripsi, harga, merek, model, tegangan):
        super().__init__(id_produk, nama, deskripsi, harga)
        self.merek = merek
        self.model = model
        self.tegangan = tegangan

File: java/program/ProdukElektronik.java
public class ProdukElektronik extends Produk {
    protected String merek;
    protected String model;
    protected int tegangan;

    public ProdukElektronik(String idProduk, String nama, String deskripsi, double harga,
                            String merek, String model, int tegangan) {
        super(idProduk, nama, deskripsi, harga);
        this.merek = merek;
        this.model = model;
        this.tegangan = tegangan;
    }

    public String getMerek() { return merek; }
    public String getModel() { return model; }
    public int getTegangan() { return tegangan; }
}
